package.json

```diff
@@ -90,13 +91,15 @@
     "escodegen": "2.0.0",
     "eslint": "7.27.0",
     "eslint-config-prettier": "8.3.0",
+    "eslint-import-resolver-babel-module": "5.3.1",
     "eslint-plugin-chai-expect": "2.2.0",
     "eslint-plugin-eslint-plugin": "3.0.3",
     "eslint-plugin-google-camelcase": "0.0.2",
     "eslint-plugin-import": "2.23.3",
     "eslint-plugin-jasmine": "4.1.2",
     "eslint-plugin-jsdoc": "35.0.0",
     "eslint-plugin-local": "1.0.0",
+    "eslint-plugin-module-resolver": "git+https://github.com/rcebulko/eslint-plugin-module-resolver.git",
```

Comment:
This one (which I have patched to fix a bug, waiting to see if the repo is active) is for eslint to require/auto-magically use alias imports when available. An example of the corresponding changes for `src/preact/component/3p-frame.js` is below. You can see that
- imports with available aliases are shortened
- module imports (`./iframe` for example) are left untouched
- un-aliased imports (`../../3p-frame`) are left untouched

```diff
-import * as Preact from '../../../src/preact';
+import * as Preact from '#preact';
 import {IframeEmbed} from './iframe';
-import {dict} from '../../core/types/object';
-import {forwardRef} from '../compat';
+import {dict} from '#core/types/object';
+import {forwardRef} from '#preact/compat';
 import {
   generateSentinel,
   getBootstrapUrl,
   getDefaultBootstrapBaseUrl,
 } from '../../3p-frame';
-import {
-  getOptionalSandboxFlags,
-  getRequiredSandboxFlags,
-} from '../../core/3p-frame';
-import {includes} from '../../core/types/string';
+import {getOptionalSandboxFlags, getRequiredSandboxFlags} from '#core/3p-frame';
+import {includes} from '#core/types/string';
 import {parseUrlDeprecated} from '../../url';
-import {sequentialIdGenerator} from '../../core/math/id-generator';
+import {sequentialIdGenerator} from '#core/math/id-generator';
 import {
   useEffect,
   useImperativeHandle,
@@ -37,7 +34,7 @@ import {
   useMemo,
   useRef,
   useState,
-} from '../../../src/preact';
+} from '#preact';
```